.github/workflows/publish.yml

```diff
@@ -0,0 +1,31 @@
+# name: 🚀 Release
+
+# on:
+#   push:
```

Comment:
We could set this one up to dispatch manually instead

Reply:
True, that's a good idea!